pywbem_mock/_dmtf_cim_schema.py

```diff
@@ -421,6 +443,58 @@ class mof compiles
             :term:`string`: Valid MOF containing pragma statements for
             all of the classes in `schema_classes`.
 
+        Raises:
+            ValueError: If any of the classnames in `schema_classes` are not in
+            the DMTF CIM schema installed
+        """
+        return DMTFCIMSchema.static_build_schema_mof(
+            schema_classes, self.schema_mof_file)
+
+    @staticmethod
+    def static_build_schema_mof(schema_classes, schema_mof_pragma_file):
+        """
+        Build a string that includes the ``#include pragma`` statements for the
+        DMTF schema CIM classes defined in `schema_classes` using the DMTF CIM
+        schema defined by this object.
```

Comment:
This method can be used on any CIM schema, not just the DMTF CIM schema. I suggest to change the docstring of this method accordingly.

Reply:
Tried to remove the DMTF everywhere but to be honest there are only the DMTF schema

Reply:
ok. good enough.